### ISSUE TITLE
"TypeError: t is not a function" when building with create-react-app

### ISSUE THREAD

**Issue by nlucero** [didierfranc/react-waterfall#56]
I'm using `react-waterfall` on a project bootstrapped with `create-react-app`. When you generate a production build running `npm run build` the output is broken. When you access to the app on your browser, this error pops up on the console:

```
TypeError: t is not a function
    at react-waterfall.js:1
    at Array.map (<anonymous>)
    at new o (react-waterfall.js:1)
    at constructClassInstance (react-dom.production.min.js:111)
    at beginWork (react-dom.production.min.js:138)
    at a (react-dom.production.min.js:176)
    at i (react-dom.production.min.js:176)
    at O (react-dom.production.min.js:182)
    at b (react-dom.production.min.js:181)
    at y (react-dom.production.min.js:181)
```

I'm using version `3.0.6`.

Any ideas?

**Comment by didierfranc** [comment]
What is the version of react you're using ?

**Comment by nlucero** [comment]
`"react": "^16.3.2"`

**Comment by nlucero** [comment]
I'll upload a repo where the problem can be reproduced shortly.

**Comment by didierfranc** [comment]
Devtools connection is included with v4 of `react-waterfall`, I think you won't encounter that kind of problem with it can you test it ?

**Comment by nlucero** [comment]
Can confirm it's not happening on `v4`. Since this is something soon to be deprecated, I'm gonna close this issue and ditch this dev tools until `v4` comes out.

**Comment by nlucero** [comment]
In case this is useful for someone with the same problem, the problem itself was on this piece of code taken from the old docs:

```
import { initStore } from 'react-waterfall';
import devtools from 'react-waterfall-redux-devtools-middleware';

const isProd = process.env.NODE_ENV === 'production';

const { ... } = initStore(store, !isProd && devtools())
```

If `process.env.NODE_ENV` is not `production` everything works fine because `!isProd && devtools()` evaluates to whatever `devtools()` returns. But if`process.env.NODE_ENV` is `production`, the expression `!isProd && devtools()` evaluates to `false` passing that as the second argument to `initStore`, producing unexpected results.

**Comment by elisherer** [comment]
Also, I tried to fix the docs in #26, but that's irrelevant now.
